README.md

```diff
@@ -183,6 +203,49 @@ var ${2:Widget} = document.registerElement('${3:my-widget}', {
 </html>
 ```
 
+### [phs] HTML template with Web Components polyfill load when not supported by browser
+
+```html
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+
+  <title>${1}</title>
+  <meta name="description" content="${2}">
+
+  <!-- Mobile -->
+  <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
+
+  <!-- Chrome / Android -->
+  <meta name="mobile-web-app-capable" content="yes">
+  <meta name="theme-color" content="black">
+  <link rel="icon" href="icon.png">
+
+  <!-- Safari / iOS -->
+  <meta name="apple-mobile-web-app-capable" content="yes">
+  <meta name="apple-mobile-web-app-status-bar-style" content="black">
+  <link rel="apple-touch-icon-precomposed" href="apple-touch-icon.png">
+
+  <!-- Check if Web Components are supported-->
+  <script>
```

Comment:
I think this is ok. I usually do this in a separate JS file but that's not a requirement. It's good to give it to folks in an easy to use format like this as well.